<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageDiskManagement</class>
 <widget class="QWidget" name="PageDiskManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TabBar" name="widget_tab" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="DiskSetting" name="pageDiskSetting"/>
     <widget class="StorageGeneralSettings" name="pageStorageGeneralSetting"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabBar</class>
   <extends>QWidget</extends>
   <header>tabbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiskSetting</class>
   <extends>QWidget</extends>
   <header>disksetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StorageGeneralSettings</class>
   <extends>QWidget</extends>
   <header>StorageGeneralSettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
